### PR TITLE
RF-1.0.0.3.1

### PR DESCRIPTION
**Pull request by EtricKombat-zz**
To raise a new Pull Request, the following prerequisites need to be met. Please tick before proceeding:

- [ ] a conversation was held in the appropriate (https://chat.whatsapp.com/El6d3aAc6pYLEDEdQPVABY) channel.
- [ ] a conversation was held in the appropriate [Slack Group](https://kochiorgbook.slack.com/) channel.
- [x] a Github Issue was opened for this feature / bug.
- [ ] test coverage was added .

This addresses **RF-1.0.0.3.1**

- Display version of KOBman, should be enabled in long form and short form

- All the 'sudo' references has been removed

### CODE REVIEW

File: dist/get.kobman.io [hunk 1]
```diff
@@ -1,13 +1,14 @@
 #!/bin/bash
 
-sudo dpkg --configure -a
 #Install: stable
 
 # Global variables
 export KOBMAN_VERSION="0.01"
 KOBMAN_PLATFORM=$(uname)
 export KOBMAN_SERVICE="https://raw.githubusercontent.com"
```
Comment (panickervinod) [inline dist/get.kobman.io:8]: Suggested change : 

export KOBMAN_SERVICE="https://raw.githubusercontent.com"/${KOBMAN_NAMESPACE}/KOBman/${KOBMAN_DIST_BRANCH}/dist/

File: dist/get.kobman.io [hunk 2]
```diff
@@ -1,13 +1,14 @@
 #!/bin/bash
 
-sudo dpkg --configure -a
 #Install: stable
 
 # Global variables
 export KOBMAN_VERSION="0.01"
```
Comment (panickervinod) [inline dist/get.kobman.io:6]: change to. ,  
export KOBMAN_VERSION=${KOBMAN_VERSION:-"0.0.1"}

@asa1997 Also update in the tmpl get.kobman.io as  export KOBMAN_VERSION=${KOBMAN_VERSION:-@KOBMAN_VERSION@}

File: dist/get.kobman.io [hunk 3]
```diff
@@ -187,7 +184,7 @@ echo "kobman_colour_enable=true" >> "$kobman_config_file"
 echo "Download script archive..."
 
 # once move to kobman namespace needs to update kobman-latest.zip 
-curl --location --progress-bar "${KOBMAN_SERVICE}/${KOBMAN_NAMESPACE}/KOBman/master/dist/kobman-latest.zip" > "$kobman_zip_file"
+curl --location --progress-bar "${KOBMAN_SERVICE}/${KOBMAN_NAMESPACE}/KOBman/${KOBMAN_DIST_BRANCH}/dist/kobman-latest.zip" > "$kobman_zip_file"
```
Comment (panickervinod) [inline dist/get.kobman.io:187]: should be ${KOBMAN_SERVICE}/kobman-latest.zip based on the change in line 8

File: dist/get.kobman.io [hunk 4]
```diff
@@ -20,14 +21,11 @@ kobman_src_folder="${KOBMAN_DIR}/src"
 kobman_tmp_folder="${KOBMAN_DIR}/tmp"
 kobman_stage_folder="${kobman_tmp_folder}/stage"
 kobman_zip_file="${kobman_tmp_folder}/kobman-${KOBMAN_VERSION}.zip"
-kobman_zip_tests="${kobman_tmp_folder}/kobman-test.zip"
-kobman_tests_folder="${KOBMAN_DIR}/tests"
 kobman_env_folder="${KOBMAN_DIR}/envs"
 kobman_stage_folder="${kobman_tmp_folder}/stage"
 kobman_etc_folder="${KOBMAN_DIR}/etc"
 kobman_var_folder="${KOBMAN_DIR}/var"
 kobman_archives_folder="${KOBMAN_DIR}/archives"
```
Comment (panickervinod) [inline dist/get.kobman.io:28]: Where is this being used?

File: dist/get.kobman.io [hunk 5]
```diff
@@ -20,14 +21,11 @@ kobman_src_folder="${KOBMAN_DIR}/src"
 kobman_tmp_folder="${KOBMAN_DIR}/tmp"
 kobman_stage_folder="${kobman_tmp_folder}/stage"
 kobman_zip_file="${kobman_tmp_folder}/kobman-${KOBMAN_VERSION}.zip"
-kobman_zip_tests="${kobman_tmp_folder}/kobman-test.zip"
-kobman_tests_folder="${KOBMAN_DIR}/tests"
 kobman_env_folder="${KOBMAN_DIR}/envs"
 kobman_stage_folder="${kobman_tmp_folder}/stage"
```
Comment (panickervinod) [inline dist/get.kobman.io:25]: where is stage, archives folder used?

File: src/main/bash/envs/kobman-kobregistery.sh [hunk 6]
```diff
@@ -1,28 +1,27 @@
 #!/bin/bash 
 
 
-
 function __kobman_install_kobregistery
```
Comment (panickervinod) [inline src/main/bash/envs/kobman-kobregistery.sh:4]: should be __kobman_install_kobregistry

File: dist/get.kobman.io (hunk 2 above)
Comment (asa1997) [reply dist/get.kobman.io:6]: Sure Vinod

File: src/main/bash/envs/kobman-tobdflow.sh [hunk 7]
```diff
@@ -2,13 +2,13 @@
 
 function __kobman_development_greenlight_dir {
```
Comment (panickervinod) [inline src/main/bash/envs/kobman-tobdflow.sh:3]: What does the function name __kobman_development_greenlight_dir suggest?

File: src/main/bash/envs/kobman-kobman.sh [hunk 8]
```diff
@@ -2,12 +2,12 @@
 function __kobman_development_kobman_dir {
```
Comment (panickervinod) [inline src/main/bash/envs/kobman-kobman.sh:3]: Why do we need a seperate dir function? what does the name signify?

File: dist/get.kobman.io (hunk 3 above)
Comment (EtricKombat-zz) [reply dist/get.kobman.io:187]: This has been modified as instructed , but the download is not happening as KOBMAN_SERVICE is not getting populated with the expected value.

File: dist/get.kobman.io (hunk 2 above)
Comment (EtricKombat-zz) [reply dist/get.kobman.io:6]: Modified

File: dist/get.kobman.io (hunk 4 above)
Comment (EtricKombat-zz) [reply dist/get.kobman.io:28]: Tests references has been removed

File: src/main/bash/envs/kobman-kobregistery.sh (hunk 6 above)
Comment (EtricKombat-zz) [reply src/main/bash/envs/kobman-kobregistery.sh:4]: modified

File: src/main/bash/envs/kobman-kobman.sh (hunk 8 above)
Comment (EtricKombat-zz) [reply src/main/bash/envs/kobman-kobman.sh:3]: This function was created as part of the requirement in which the environment specific folders has to created @ home directory (outside .kobman)

File: dist/get.kobman.io (hunk 5 above)
Comment (EtricKombat-zz) [reply dist/get.kobman.io:25]: Once these stage, archives are commented the kobman-latest.zip download is not happening

File: src/main/bash/envs/kobman-tobdflow.sh (hunk 7 above)
Comment (EtricKombat-zz) [reply src/main/bash/envs/kobman-tobdflow.sh:3]: This function name can be modified as per suggestions, the name is suppose to give the idea that "greenlight development environment directory " creation is happening here

**Pull request closed without merging**